index.html

```diff
@@ -84,5 +84,162 @@ <h1>
+            </ul>
+            <button class="popup">See Project</button>
+          </div>
+        </div>
+        <div class="item">
+          <div class="placeholder"></div>
+          <div class="text">
+            <h3>To-do List</h3>
+            <ul class="skillsbar">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button class="popup">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="About">
+      <div class="aboutme-text">
+        <div class="head-ele">
+          <h1 class="aboutme-header">About Myself</h1>
+          <div class="desktop bg-img-1"></div>
+          <div class="desktop bg-img-2"></div>
+        </div>
+        <p class="aboutme-p">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <span class="aboutme-p1"> Let's connect </span>
+        <div class=".default-row-spacer">
+          <ul class="social-1">
+            <li>
+              <a href="#"><img src="./images/twitter.jpg" alt="twitter" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/Linkedinicon.jpg" alt="linkedIn"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/medium.jpg" alt="microverse" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/github.jpg" alt="github" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/angellist.jpg" alt="angellist"
+              /></a>
+            </li>
+          </ul>
+        </div>
+        <div class="head-ele">
+          <button class="about-button">Get My Resume</button>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="language-dropdown">
+        <span>Languages</span>
+        <i class="fal fa-1x fa-angle-down"></i>
+      </div>
+      <div>
+      <div class="language-list">
+        <img class="list-images-1" src="./images/Ellipse-1.png" alt="angellist" />
+        <span>JavaScript</span>
+      </div>
+      <div class="language-list">
+        <img class="list-images-1" src="./images/Ellipse-2.png" alt="angellist" />
+        <span>HTML</span>
+      </div>
+      <div class="language-list">
+        <img class="list-images-1" src="./images/Ellipse-3.png" alt="angellist" />
+        <span>CSS</span>
+      </div>
+    </div>
+    <div class="main-dropdown">
+      <span>Frameworks</span>
+      <i class="fal fa-1x fa-angle-right"></i>
+    </div>
+    <div class="main-dropdown">
+      <span>Skills</span>
+      <i class="fal fa-1x fa-angle-right"></i>
+    </div>
```

Comment:
- [ ] There is an issue with the skills cards/section because these are different from the chosen template. I would suggest updating these to match the Figma Design as it is required :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/143493981-e362919c-a280-407b-9f05-2706223aa0e1.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/143494005-794b5cde-8fbc-49d1-b8e9-f892861b1bcb.png)

**Hint: I think the skills cards could be inside of the "About" section** :wink: